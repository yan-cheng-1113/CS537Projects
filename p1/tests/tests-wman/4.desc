No man page in specified section - should exit returning 0 and print message
