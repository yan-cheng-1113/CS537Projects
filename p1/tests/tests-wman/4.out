No manual entry for wman in section 2
